subgraphs/inverse-finance/README.md

```diff
@@ -1,40 +1,44 @@
 # Inverse Finance Lending Protocol Subgraph
+
+## Useful Links
+
+- https://www.inverse.finance/anchor
+- https://www.inverse.finance/transparency/overview
+- https://docs.inverse.finance/inverse-finance/about-inverse
+
 ## Calculation Methodology v0.0.1
 
 ### Total Value Locked (TVL) USD
 
 Sum across all Pools:
 
-`Insert Calculation`
+`Sum of Deposits on Anchor`
 
-<Add notes to consider if any - delete if none>
+This includes INV staking
```

Comment:
Interesting. Why does this include INV staking? I thought we are not including staked tokens in TVL.